MediaContent/Model/ExtractAssetFromContent.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContent\Model;
+
+use Magento\Framework\Exception\IntegrationException;
+use Magento\MediaContentApi\Api\ExtractAssetFromContentInterface;
+use Magento\MediaGalleryApi\Api\Data\AssetInterface;
+use Magento\MediaGalleryApi\Model\Asset\Command\GetByPathInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Used for extracting media asset list from a media content by the search pattern.
+ */
+class ExtractAssetFromContent implements ExtractAssetFromContentInterface
```

Comment:
```suggestion
class GetAssetsUsedInContent implements ExtractAssetFromContentInterface
```

Reply:
`GetAssetsUsedInContent` already exists and it gets data from the data storage. Should we change this?